Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -55,3 +55,8 @@ trait-name-Foreigner = Foreigner
 trait-description-Foreigner =
     For one reason or another you do not speak this station's primary language.
     Instead, you have a translator issued to you that only you can use.
+
+trait-name-LightStep = Light Step
+trait-description-LightStep =
+    You move with a quiet step, making it easier for you to elude detection.
+    Your footsteps are roughly 50% quieter.
```

Comment:
```suggestion
```

Reply:
I think the trait description would be too vague without that line, as I wrote the first line as sort of a flavor text.

Reply:
Resolved with [5b8c2d8](https://github.com/Simple-Station/Einstein-Engines/pull/658/commits/5b8c2d8c6bd9c99cd165bdf3a65d053804c3eaa1)